packages/astro-prism/src/internal.ts

```diff
@@ -179,10 +181,49 @@ export function addAstro(Prism) {
 		}
 	};
 
-	Prism.hooks.add('after-tokenize', function (env) {
+	Prism.hooks.add('after-tokenize', function (env: any) {
 		if (env.language !== 'astro') {
 			return;
 		}
 		walkTokens(env.tokens);
 	});
 }
+
+const languageMap = new Map([['ts', 'typescript']]);
```

Comment:
Can you split this code into a different module? I think it would make it easier to understand what each module is for. The above plugin was modeled after how Prism plugins are written (see https://github.com/PrismJS/prism/blob/master/plugins/file-highlight/prism-file-highlight.js for example). Maybe this file should be called `plugin.ts`. No problem with any of the code in this module, just suggesting some organization.